javadoc build errors under jdk17.0.7

While testing out a new linux system, I tried building the current cdi master and found a few errors due to javadoc issues. This builds fine under jdk11, but the errors seem correct so I assume the checking has gotten stricter.

These are the 5 errors I see:
```
/home/starksm/Dev/Jakarta/cdi/api/src/main/java/jakarta/decorator/package-info.java:35: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Decorated types</h3>
   ^
/home/starksm/Dev/Jakarta/cdi/api/src/main/java/jakarta/enterprise/context/package-info.java:32: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Built-in scopes</h3>
   ^
/home/starksm/Dev/Jakarta/cdi/api/src/main/java/jakarta/enterprise/event/package-info.java:40: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Event objects and event types</h3>
   ^
/home/starksm/Dev/Jakarta/cdi/api/src/main/java/jakarta/enterprise/inject/spi/package-info.java:31: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>The <code>BeanManager</code> object</h3>
home/starksm/Dev/Jakarta/cdi/api/src/main/java/jakarta/enterprise/event/Observes.java:109: error: unexpected end tag: </p>
 * </p>
```

There are also a ton of warnings about no comments that can be addressed later.

--- a/.github/workflows/ci-actions.yml
+++ b/.github/workflows/ci-actions.yml
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         java:
-          - 11
+          - [ '11', '17' ]
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK ${{ matrix.java }}

--- a/.github/workflows/ci-actions.yml
+++ b/.github/workflows/ci-actions.yml
@@ -11,8 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java:
-          - [ '11', '17' ]
+        java: [ '11', '17' ]
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK ${{ matrix.java }}

--- a/api/src/main/java/jakarta/decorator/package-info.java
+++ b/api/src/main/java/jakarta/decorator/package-info.java
@@ -32,7 +32,7 @@
  * for which interceptors are optimized. Decorators are called 
  * after interceptors.</p>
  * 
- * <h3>Decorated types</h3>
+ * <h2>Decorated types</h2>
  * 
  * <p>The set of decorated types of a decorator includes all 
  * bean types of the managed bean that are Java interfaces, 
@@ -49,7 +49,7 @@
  * <p>A decorator may be an abstract class, and is not required to 
  * implement every method of every decorated type.</p>
  * 
- * <h3>Delegate injection points</h3>
+ * <h2>Delegate injection points</h2>
  * 
  * <p>All decorators have a 
  * {@linkplain jakarta.decorator.Delegate delegate injection point}.  
@@ -60,7 +60,7 @@
  * extend every decorated type. A decorator is not required to 
  * implement the type of the delegate injection point.</p>
  * 
- * <h3>Enabled decorators</h3>
+ * <h2>Enabled decorators</h2>
  *  
  * <p>By default, a bean archive has no enabled decorators. A 
  * decorator must be explicitly enabled by listing its bean class 

--- a/api/src/main/java/jakarta/enterprise/context/package-info.java
+++ b/api/src/main/java/jakarta/enterprise/context/package-info.java
@@ -29,7 +29,7 @@
  * bean</li>
  * </ul>
  * 
- * <h3>Built-in scopes</h3>
+ * <h2>Built-in scopes</h2>
  * 
  * <p>The following built-in scopes are provided:
  * {@link jakarta.enterprise.context.Dependent &#064;Dependent},
@@ -57,7 +57,7 @@
  * method invocations or to asynchronous processes such as JMS message 
  * listeners or EJB timer service timeouts.</p>
  * 
- * <h3>Normal scopes and pseudo-scopes</h3>
+ * <h2>Normal scopes and pseudo-scopes</h2>
  * 
  * <p>Most scopes are <em>normal scopes</em>. Normal scopes are declared 
  * using {@link jakarta.enterprise.context.NormalScope &#064;NormalScope}. 
@@ -80,7 +80,7 @@
  * {@link jakarta.enterprise.context.Dependent &#064;Dependent} and
  * {@link jakarta.inject.Singleton &#064;Singleton} pseudo-scopes.</p>
  * 
- * <h3>Contextual and injected reference validity</h3>
+ * <h2>Contextual and injected reference validity</h2>
  * 
  * <p>A reference to a bean obtained from the container via {@linkplain 
  * jakarta.enterprise.inject.Instance programmatic lookup} is called a 

--- a/api/src/main/java/jakarta/enterprise/event/Observes.java
+++ b/api/src/main/java/jakarta/enterprise/event/Observes.java
@@ -105,8 +105,8 @@ public @interface Observes {
      * 
      * <p>
      * By default, the observer method is notified when the event is fired.
-     * @return phase of the transaction
      * </p>
+     * @return phase of the transaction
      */
     public TransactionPhase during() default TransactionPhase.IN_PROGRESS;
 }

--- a/api/src/main/java/jakarta/enterprise/event/package-info.java
+++ b/api/src/main/java/jakarta/enterprise/event/package-info.java
@@ -37,7 +37,7 @@
  * <p>The {@link jakarta.enterprise.event.Event} interface is used to 
  * fire events.</p>
  * 
- * <h3>Event objects and event types</h3>
+ * <h2>Event objects and event types</h2>
  * 
  * <p>The event object acts as a payload, to propagate state from 
  * producer to consumer. An event object is an instance of a concrete 
@@ -47,13 +47,13 @@
  * interfaces of the runtime class of the event object. An event type 
  * may not contain a type variable.</p>
  * 
- * <h3>Event qualifiers</h3>
+ * <h2>Event qualifiers</h2>
  * 
  * <p>The event qualifiers act as topic selectors, allowing the consumer 
  * to narrow the set of events it observes. An event qualifier may be an
  * instance of any {@linkplain jakarta.inject.Qualifier qualifier type}.</p>
  * 
- * <h3>Observer methods</h3>
+ * <h2>Observer methods</h2>
  * 
  * <p>An {@linkplain jakarta.enterprise.event.Observes observer method} 
  * allows the application to receive and respond synchronously to event notifications.

--- a/api/src/main/java/jakarta/enterprise/inject/spi/package-info.java
+++ b/api/src/main/java/jakarta/enterprise/inject/spi/package-info.java
@@ -28,7 +28,7 @@
  * metadata from some other source</li>
  * </ul>
  * 
- * <h3>The <code>BeanManager</code> object</h3>
+ * <h2>The <code>BeanManager</code> object</h2>
  * 
  * <p>Portable extensions sometimes interact directly with the container 
  * via programmatic API call. The interface 
@@ -36,7 +36,7 @@
  * for obtaining contextual references for beans, along with many other 
  * operations of use to portable extensions.</p>
  * 
- * <h3>Container lifecycle events</h3>
+ * <h2>Container lifecycle events</h2>
  * 
  * <p>During the application initialization process, the container fires 
  * a series of {@linkplain jakarta.enterprise.event events}, allowing 
@@ -51,7 +51,7 @@
  * {@link jakarta.enterprise.inject.spi.AfterDeploymentValidation} and
  * {@link jakarta.enterprise.inject.spi.BeforeShutdown}.</p>
  * 
- * <h3>Interfaces representing enabled beans</h3>
+ * <h2>Interfaces representing enabled beans</h2>
  * 
  * <p>The interfaces 
  * {@link jakarta.enterprise.inject.spi.Bean},
@@ -83,7 +83,7 @@
  * observing the event type
  * {@link jakarta.enterprise.inject.spi.ProcessObserverMethod}.</p>
  * 
- * <h3>Alternate metadata sources</h3>
+ * <h2>Alternate metadata sources</h2>
  * 
  * <p>A portable extension may provide an alternative metadata 
  * source, such as configuration by XML.</p>
@@ -104,7 +104,7 @@
  * {@linkplain jakarta.enterprise.inject.spi.ProcessAnnotatedType#setAnnotatedType(AnnotatedType)
  * wrapping} the {@link jakarta.enterprise.inject.spi.AnnotatedType}.</p>
  * 
- * <h3><code>Producer</code> and <code>InjectionTarget</code></h3>
+ * <h2><code>Producer</code> and <code>InjectionTarget</code></h2>
  * 
  * <p>The interfaces {@link jakarta.enterprise.inject.spi.Producer} and
  * {@link jakarta.enterprise.inject.spi.InjectionTarget} abstract the 

